src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -857,5 +896,9 @@ ul {
     font-size: $px-16;
     font-weight: normal;
   }
+
+  .v-input input {
+    color: $gray9
+  }
```

Comment:
All these styles should be right in the BaseAddress component.

Please create a new ticket if you don't have time in your current ticket to update this.

Reply:
Created!

https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/11547